src/VerifyWindowsHelloController.js

```diff
@@ -121,7 +124,8 @@ function (Okta, FormController, FormType, webauthn, Spinner, FooterSignout, Wind
       },
 
       postRender: function () {
-        if (this.options.appState.get('factors').length === 1 && !this.model.get('__enrollmentState__')) {
+        if (this.options.appState.get('factors').length === 1 && !this.model.get('__autoTriggered__')) {
```

Comment:
The bug was due to the recursion in the logic:
1. The view was added and calls `render()`
2. `render()` calls `postRender()`
3. If Windows Hello is the only factor, `postRender()` calls the `model.save()`
4. `model.save()` triggers the `request` event, that changes the title and calls `render()`.
5. See step 2

To check if it is the very first `postRender` I added the `__autoTriggered__ ` flag.